Tests/New-HipchatUser.Tests.ps1

```diff
@@ -1,16 +1,32 @@
 ﻿Import-Module PHipChatAdmin
```

Comment:
PHipChatAdmin should be HipChatAdmin. The P must've been a typo that I made. Fixed this yesterday in a separate PR but if you could fix it here as well that would be awesome!